db/migrate/20220509104200_create_goods_nomenclature_export_function.rb

```diff
@@ -0,0 +1,114 @@
+Sequel.migration do
+  up do
+    Sequel::Model.db.run('CREATE SCHEMA IF NOT EXISTS "public"')
```

Comment:
Is this necessary - wont `public` exist already?

Reply:
Yes, I'll remove it.